ui/dashboards/src/components/Panel/PanelHeader.tsx

```diff
@@ -96,14 +97,25 @@ export function PanelHeader({ id, title, description, editHandlers, isHovered, s
           {title}
         </Typography>
       }
-      action={action}
+      action={
+        <HeaderActionWrapper direction="row" spacing={0.25} alignItems="center">
```

Comment:
Putting this wrapper down here, so we don't have to duplicate it above. It doesn't seem to hurt anything to have an empty element here when the actions are `undefined`.